HTTPServerProject/Program.cs

```diff
@@ -19,24 +21,26 @@ public Server(TcpClient tcpc)
 
         public void Conversation()
         {
+            NetworkStream stream = client.GetStream();
+            StreamWriter writer = new StreamWriter(stream);
+            MyStreamReader reader = new MyStreamReader(stream);
+
             Console.WriteLine("Connection accepted.");
 
-            var stream = client.GetStream();
-            var reader = new MyStreamReader(stream: stream);
-            var writer = new StreamWriter(stream);
+            Header header = new Header(reader);
+            var initialLine = header.GetLine();
+            var rHeader = header.GetHeaders();
 
-            var input = reader.ReadLine()!;
+            Body body = new Body(reader);
+            var input = body.GetBody();
```

Comment:
Are you using the `reader` to read the part of the request that has headers, and then using the `reader` to read the part that has the body, because of the issues you've been having around using `readLine` for the body?